tools/tertiary-analysis/scanpy/scanpy-neighbours.xml

```diff
@@ -15,7 +15,7 @@ PYTHONIOENCODING=utf-8 scanpy-neighbours.py
     #if $settings.default == "false"
         -N '${settings.n_neighbours}'
         -m '${settings.method}'
-        -s '${settings.seed}'
+        -s '${settings.random_seed}'
```

Comment:
Good, this might have slipped. Can you please bump the galaxy version of the tool (+galaxy<value++>).